packages/starlight/index.ts

```diff
@@ -45,12 +45,7 @@ export default function StarlightIntegration(opts: StarlightUserConfig): AstroIn
 							// Configure Shiki theme if the user is using the default github-dark theme.
 							config.markdown.shikiConfig.theme !== 'github-dark' ? {} : { theme: 'css-variables' },
 					},
-					build: { inlineStylesheets: 'auto' },
-					experimental: {
-						assets: true,
-						// @ts-ignore - Needed for older versions of Astro, but an error since astro@2.6.0
-						inlineStylesheets: 'auto',
-					},
+					scopedStyleStrategy: 'where',
```

Comment:
For now I think the safest is to set this to `'where'`, which was the Astro v2 default. The new default breaks some Starlight styling due to shifts in relative CSS specificity between global and component CSS. Those same shifts likely make it harder for Starlight users to override components, so I think `'where'` is the right choice.

Long term, I think we’ll want to move towards using `@layer starlight {}` for our built-in styles to help make it easier for users to override things with their CSS without worrying about specificity. At that point, we’d probably be safe to switch back to the Astro default and refactor a few internal styles where things clash.